Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml

```diff
@@ -75,77 +93,48 @@
 - type: entity
   parent: ClothingHeadsetAlt
   id: ClothingHeadsetAltSecurity
-  name: security over-ear headset
+  name: security command over-ear headset
   components:
   - type: Headset
-    channels:
-    - Command
-    - Common
-    - Security
     defaultChannel: Security
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyHOS
   - type: Sprite
     sprite: Clothing/Ears/Headsets/security.rsi
   - type: Clothing
     sprite: Clothing/Ears/Headsets/security.rsi
 
-- type: entity
-  parent: ClothingHeadsetAlt
-  id: ClothingHeadsetAltEngineering
-  name: engineering over-ear headset
-  components:
-  - type: Headset
-    channels:
-    - Command
-    - Common
-    - Engineering
-    defaultChannel: Engineering
-  - type: Sprite
-    sprite: Clothing/Ears/Headsets/engineering.rsi
-  - type: Clothing
-    sprite: Clothing/Ears/Headsets/engineering.rsi
- 
 - type: entity
   parent: ClothingHeadsetAlt
   id: ClothingHeadsetAltScience
-  name: science over-ear headset
+  name: rd over-ear headset
   components:
   - type: Headset
-    channels:
-    - Command
-    - Common
-    - Science
     defaultChannel: Science
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyRD
   - type: Sprite
     sprite: Clothing/Ears/Headsets/science.rsi
   - type: Clothing
     sprite: Clothing/Ears/Headsets/science.rsi
 
-- type: entity
-  parent: ClothingHeadsetAlt
-  id: ClothingHeadsetAltCargo
-  name: cargo over-ear headset
-  components:
-  - type: Headset
-    channels:
-    - Command
-    - Common
-    - Supply
-    defaultChannel: Supply
-  - type: Sprite
-    sprite: Clothing/Ears/Headsets/cargo.rsi
-  - type: Clothing
-    sprite: Clothing/Ears/Headsets/cargo.rsi
-
 - type: entity
   parent: ClothingHeadsetAlt
   id: ClothingHeadsetAltSyndicate
-  name: syndicate over-ear headset
-  description: An updated, modular syndicate intercom that fits over the head and takes encryption keys. Protects ears from flashbangs.
+  name: blood-red over-ear headset
+  description: An updated, modular syndicate intercom that fits over the head and takes encryption keys (there are 4 of them). Should protect ears from flashbangs.
```

Comment:
This description didn't need a change.

Reply:
But it isn't protecting from flashbangs

Reply:
Then delete that part entirely, don't mislead players

Reply:
Already, thank you